business-central-parent/business-central-webapp/pom.xml

```diff
@@ -2770,8 +2759,6 @@
 
             <!-- TODO kogito needs refactoring out of Business Central -->
             <compileSourcesArtifact>org.kie.workbench:kie-wb-common-kogito-client</compileSourcesArtifact>
```

Comment:
Why can't we remove this too?

Reply:
yeah @domhanak good catch :) I really wanted to also try to drop this module, but this module is not inside stunner codebase itself (it's part of the repository common modules, see https://github.com/kiegroup/kie-wb-common/tree/master/kie-wb-common-kogito), and once trying to drop it locally I found other usages as well.

So finally I decided not to add more complexity to this PR, otherwise the refactor would take longer and would result even bigger. I think it can be done in a saf and easier way in a second iteration.

Anyway good catch, it's somethign I've also in mind continuously :)

Thanks!